monitor/newrelic.py

```diff
@@ -0,0 +1,24 @@
+from monitor.metric import UpcloudMetrics
+
+from newrelic_telemetry_sdk import GaugeMetric, MetricClient
+
+
+class NewRelicClient:
+    def __init__(self, license_key: str):
+        self.__client = MetricClient(license_key)
+
+    def send(self, metrics: UpcloudMetrics):
+        batch = []
+
+        for metric in metrics.metrics:
+            for record in metric.records:
+                batch.append(
```

Comment:
Any reason that we don't use `newrelic.agent.record_custom_metric`?

Reply:
Custom metrics don't allow to add tags. We're using them to differentiate each db server.

Reply:
We're still trying this out though and see how it's shown in newrelic. it's weird that custom  metrics don't support tags though.

Reply:
by db server, do you mean staging and prod? Currently staging uses a completely different newrelic account.

Reply:
no, master and replicas

Reply:
`newrelic.agent.record_custom_metric` doesn't allow us to add metadata to the metric (such as the host).

In order to add that metadata, we are forced to do it as `Custom/name/label`, and it only supports one level of metadata.

The NewRelic Telemetry SDK allows us to define the variable type (Gauge in our case) and add any extra information to the metric we may need. For now it's only the server name, but could be expanded to more, should we need it.

Reply:
Ok. It's working and it allows to have metrics with dimensions per host. :tada: